backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQHealthCheckTest.java

```diff
@@ -25,23 +25,23 @@
 
 import org.apache.james.core.healthcheck.Result;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+import com.rabbitmq.client.ConnectionFactory;
+
 @ExtendWith(DockerRabbitMQExtension.class)
 class RabbitMQHealthCheckTest {
     private RabbitMQHealthCheck healthCheck;
 
     @BeforeEach
     void setUp(DockerRabbitMQ rabbitMQ) throws Exception {
         URI amqpUri = URI.create("amqp://" + rabbitMQ.getHostIp() + ":" + rabbitMQ.getPort());
-        URI managementUri = URI.create("http://" + rabbitMQ.getHostIp() + ":" + rabbitMQ.getAdminPort());
-
+        ConnectionFactory connectionFactory = new ConnectionFactory();
```

Comment:
Why not directly `new ConnectionFactory(amqpUri)`?

Reply:
Because such constructor doesn't exist.